rpcs3/rpcs3qt/pad_settings_dialog.cpp

```diff
@@ -336,29 +342,28 @@ void pad_settings_dialog::InitButtons()
 		RepaintPreviewLabel(ui->preview_stick_right, value, ui->slider_stick_right->size().width(), rx, ry);
 	});
 
-	connect(ui->b_led, &QPushButton::clicked, [=]()
+	// Open LED settings
+	connect(ui->b_led_settings, &QPushButton::clicked, [=]()
 	{
-		QColor led_color(m_handler_cfg.colorR, m_handler_cfg.colorG, m_handler_cfg.colorB);
-		if (ui->b_led->property("led").canConvert<QColor>())
-		{
-			led_color = ui->b_led->property("led").value<QColor>();
-		}
-		QColorDialog dlg(led_color, this);
-		dlg.setWindowTitle(tr("LED Color"));
-		if (dlg.exec() == QColorDialog::Accepted)
-		{
-			const QColor newColor = dlg.selectedColor();
-			m_handler->SetPadData(m_device_name, 0, 0, newColor.red(), newColor.green(), newColor.blue());
-			ui->b_led->setIcon(gui::utils::get_colorized_icon(QIcon(":/Icons/controllers.png"), Qt::black, newColor));
-			ui->b_led->setProperty("led", newColor);
-		}
+		pad_led_settings_dialog dialog(m_handler_cfg.colorR, m_handler_cfg.colorG, m_handler_cfg.colorB, static_cast<bool>(m_handler_cfg.led_low_battery_blink), static_cast<bool>(m_handler_cfg.led_battery_indicator), m_handler_cfg.led_battery_indicator_brightness, this);
+		// Execute regardless of accept or reject since there's nothing to do, the window is handled by a slot
```

Comment:
The comment should be removed as the connect is pretty much self explanatory
```suggestion
		connect(&dialog, &pad_led_settings_dialog::pass_led_settings, this, &pad_settings_dialog::apply_led_settings);
```